### PR TITLE
Fix style

### PR DESCRIPTION
**Pull request by mastemm**

### CODE REVIEW

File: frontend/src/components/Login/style.jsx [hunk 1]
```diff
@@ -30,13 +31,16 @@ export default styled.div`
     background: #f77777;
     border: 1px solid #ffffff;
     border-radius: 1.6rem;
+    color: white;
+    font-size: 1.2rem;
+    letter-spacing: 0.1rem;
   }
 
   @media screen and (min-width: 500px) {
     .formBox {
```
Comment (SDecambray) [inline frontend/src/components/Login/style.jsx:40]: padding : top right bottom left

**Review by ghost — CHANGES_REQUESTED** [review]
Just change the padding plz

File: frontend/src/components/Login/style.jsx [hunk 2]
```diff
@@ -30,13 +31,16 @@ export default styled.div`
...
       padding-top: 1rem;
-      padding-left: 6rem;
-      padding-right: 6rem;
+      padding-left: 7rem;
```
Comment (ghost) [inline frontend/src/components/Login/style.jsx]: padding : vertical horizontal

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
Ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**